framework/src/modules/dpos_v2/commands/delegate_registration.ts

```diff
@@ -52,6 +61,25 @@ export class DelegateRegistrationCommand extends BaseCommand {
 			};
 		}
 
+		if (context.params.delegateRegistrationFee !== DELEGATE_REGISTRATION_FEE) {
+			return {
+				status: VerifyStatus.FAIL,
+				error: new Error('Invalid delegate registration fee.'),
+			};
+		}
+
+		const balance = await this._tokenMethod.getAvailableBalance(
+			context,
+			transaction.senderAddress,
+			TOKEN_ID_FEE,
+		);
+		if (balance < DELEGATE_REGISTRATION_FEE) {
```

Comment:
`DELEGATE_REGISTRATION_FEE` is also configurable constants, so it should come from config